quick-start-with-tidb.md

```diff
@@ -6,510 +6,532 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
+    ('Central Park', 'New York, USA', 40.7829, -73.9654);
+```
 
-> **Note:**
->
-> The IP addresses of the instances are given as examples only. In your actual deployment, replace the IP addresses with your actual IP addresses.
+### Generate sample workload
 
-| Instance | Count | IP | Configuration |
-|:-- | :-- | :-- | :-- |
-| TiKV | 3 | 10.0.1.1 <br/> 10.0.1.1 <br/> 10.0.1.1 | Avoid conflict between the port and the directory |
-| TiDB | 1 | 10.0.1.1 | The default port <br/> Global directory configuration |
-| PD | 1 | 10.0.1.1 | The default port <br/> Global directory configuration |
-| TiFlash | 1 | 10.0.1.1 | The default port <br/> Global directory configuration |
-| Monitor | 1 | 10.0.1.1 | The default port <br/> Global directory configuration |
+Open two new terminal windows to generate write and read workloads through TiProxy.
 
-Other requirements for the target machine include:
+Terminal 1: Write Workload
 
-- The `root` user and its password are required
-- [Stop the firewall service of the target machine](/check-before-deployment.md#check-and-stop-the-firewall-service-of-target-machines), or open the port needed by the TiDB cluster nodes
-- Currently, the TiUP cluster supports deploying TiDB on the x86_64 (AMD64) and ARM architectures:
+Run this command to continuously insert weather readings:
 
-    - It is recommended to use CentOS 7.3 or later versions on AMD64.
-    - It is recommended to use CentOS 7.6 1810 on ARM.
+```shell
+while true; do
+    mysql -h 127.0.0.1 -P 6000 -u root -D sample_weather -vv -e "INSERT INTO readings
+    (station_id, temperature, humidity, pressure) SELECT id as station_id, ROUND(10 + (RAND() *
+    30), 1), ROUND(40 + (RAND() * 60)), ROUND(980 + (RAND() * 40)) FROM stations;"
+    sleep 0.5
+done
+```
 
-### Deploy
+Terminal 2: Read Workload
 
-> **Note:**
+Run this command to continuously query the data:
+
+```shell
+while true; do
+    mysql -h 127.0.0.1 -P 6000 -u root -D sample_weather -e "SELECT s.name, s.location,
+    r.temperature, r.humidity, r.recorded_at FROM stations s JOIN readings r ON s.id = r.station_id
+    WHERE r.recorded_at >= NOW() - INTERVAL 5 MINUTE ORDER BY r.recorded_at DESC LIMIT 5;"
+    sleep 1
+    mysql -h 127.0.0.1 -P 6000 -u root -D sample_weather -e "SELECT s.name,
+    ROUND(AVG(r.temperature), 1) as avg_temp, ROUND(AVG(r.humidity)) as avg_humidity,
+    COUNT(*) as num_readings FROM stations s JOIN readings r ON s.id = r.station_id WHERE
+    r.recorded_at >= NOW() - INTERVAL 1 HOUR GROUP BY s.id, s.name ORDER BY avg_temp DESC;"
+    sleep 3
+done
+```
+
+## Monitor your application
+
+Now that you have a running application with active read and write workloads, let's use TiDB's built-in monitoring tools to understand what's happening.
+
+### TiDB Dashboard: SQL insights
+
+Access TiDB Dashboard by checking its URL on the terminal you are running TiUP playground (typically http://127.0.0.1:2379/dashboard - user: `root`, no password) to:
```

Comment:
"http://127.0.0.1:2379/dashboard" is confusing here because this section assumes TiProxy is added. In that case, "http://127.0.0.1:60434/dashboard/" is also typical.
```suggestion
Access TiDB Dashboard by using its URL (for example, http://127.0.0.1:2379/dashboard) in the terminal where you are running TiUP playground. The default login is root with no password.
```